src/anchor/anchor.scss

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-anchor($status: primary, $underline: 'false') {
+  --_iui-anchor-popout-svg: url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'><path d='m16 0v5.4l-1.9-2-8.4 8.4-1.5-1.5 8.3-8.4-1.9-1.9m5.4 16v-9h-1v8h-14v-14h8v-1h-9v16z' /></svg>");
+
+  @include iui-anchor-status($status);
+  border-radius: $iui-border-radius;
+  text-underline-offset: 1px;
+  forced-color-adjust: none;
```

Comment:
Forces colors to not be overriden since we need to reserve the color in some cases for status.
![Screen Shot 2022-03-28 at 4 44 03 PM](https://user-images.githubusercontent.com/849817/160483832-9c871c0e-3db2-49da-9fac-91453adcc97c.png)

Without it we get this and the icon disappears:
![Screen Shot 2022-03-28 at 4 45 22 PM](https://user-images.githubusercontent.com/849817/160484042-330fa044-0333-4b80-ab05-f3208831c456.png)